test/e2e/immutable-parameter/03-uninstall.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo uninstall --instance immutable-param
```

Comment:
is this necessary? I thought kuttl is doing cleanup...

Reply:
It's not, good point. I've copied the `cli-install-uninstall` test as a base and blindly adjusted the last step :) Fixed